department-of-veterans-affairs/caseflow
# APPEALS-42458

Resolves [APPEALS-42458](https://jira.devops.va.gov/browse/APPEALS-42458)

# Description
Disables edit issues and request issue modification for remands

## Acceptance Criteria
- [ ] When selecting the disposition page for a Remand, the Edit Issue button is disabled

## Testing Plan

1. Go to [Jira Issue/Test Plan Link](https://jira.devops.va.gov/browse/JIRA-12345) or list them below
2. Generate a remand by opening a rails console and doing: `FactoryBot.create(:remand_vha_task, assigned_at: 1.minute.ago)`
3. Navigate to the VHA DR queue as an admin user (VHAADMIN) and select the task you just created
4. Verify the edit issues button is disabled
5. View the same task as a non-admin user and verify request issue modification is disabled


# Frontend
## User Facing Changes

 BEFORE|AFTER
 ---|---

## Storybook Story
*For Frontend (Presentation) Components*

# Backend
## Database Changes
*Only for Schema Changes*


## Integrations: Adding endpoints for external APIs

# Best practices
## Code Documentation Updates

## Tests
### Test Coverage
Did you include any test coverage for your code? Check below:

### Code Climate
Your code does not add any new code climate offenses? If so why?

## Monitoring, Logging, Auditing, Error, and Exception Handling Checklist

### Monitoring

### Logging

### Auditing

### Error Handling

### Exception Handling
